Packages/TasktiveLocale/Sources/TasktiveLocale/Keys.swift

```diff
@@ -18,10 +18,12 @@ extension TasktiveLocale {
         case GET_ALL_TASKS_ERROR_DESCRIPTION
         case CREATE_TASK_ERROR_DESCRIPTION
         case UPDATE_TASK_ERROR_DESCRIPTION
+        case DELETE_TASK_ERROR_DESCRIPTION
```

Comment:
This doesn't seem to be in `Locales/en.js`, if you put the locale there and run `yarn localize` it will appear here.

Reply:
fixed